### DISCUSSION TITLE
How to use on Error callback in next Image?

### DISCUSSION THREAD

**Discussion by AgentPrus** [vercel/next.js#21887] [Help]
I've faced with an issue when I have an error on loading images and I want provide a default image when an error occurs.
So when I've used HTML image tag I've done this by providing onErorr call back look code below:
``` 
const handleError = (event) => {
  event.target.src = defaultImgUrl;
}

<img
	src={url}
	onError={handleError}
/>
```

But when I'm doing the same with next Image I always getting errors that I provided the wrong default image URL. And It makes an infinite loop of errors.

**Comment by Blundert** [comment]
@AgentPrus have you found a solution to this problem?

**Comment by claide** [comment]
hello 2024
